xbmc/ServiceManager.cpp

```diff
@@ -148,6 +151,16 @@ bool CServiceManager::InitStageTwo(const CAppParamParser &params, const std::str
   m_mediaManager.reset(new CMediaManager());
   m_mediaManager->Initialize();
 
+#if !defined(TARGET_WINDOWS) && defined(HAS_DVD_DRIVE)
+  // Start Thread for DVD Mediatype detection
+  CLog::Log(LOGINFO, "start dvd mediatype detection");
+  m_DetectDVDType.reset(new MEDIA_DETECT::CDetectDVDMedia());
```

Comment:
```suggestion
  m_DetectDVDType = std::make_unique<MEDIA_DETECT::CDetectDVDMedia>();
```

Reply:
is the reason to hide `new` inside the unique ptr?